haystack/nodes/answer_generator/openai.py

```diff
@@ -87,13 +87,16 @@ def __init__(
                                   [See more information about frequency and presence penalties.](https://platform.openai.com/docs/api-reference/parameter-details)
         :param examples_context: A text snippet containing the contextual information used to generate the Answers for
                                  the examples you provide.
-                                 If not supplied, the default from OpenAI API docs is used:
-                                 `"In 2017, U.S. life expectancy was 78.6 years."`
         :param examples: List of (question, answer) pairs that helps steer the model towards the tone and answer
-                         format you'd like. We recommend adding 2 to 3 examples.
-                         If not supplied, the default from OpenAI API docs is used:
-                         `[["Q: What is human life expectancy in the United States?", "A: 78 years."]]`
-        :param stop_words: Up to four sequences where the API stops generating further tokens. The returned text does
+                         format you'd like.
+        :param instructions: Here you can initialize custom instructions as prompt. Defaults to 'Create a concise and informative answer...'
+        :param add_runtime_instructions: If you like to add the prompt instructions (the instructions around the question)
+                                         during querying or not. Defaults to using predefined prompt instructions.
+                                         If you do add instructions at runtime separate instructions and question like:
+                                         "... <instructions> ... [SEPARATOR] <question>"
+                                         Also make sure to mention "$documents" and "$query" in the <instructions>, such
+                                         that those will be replaced in correctly.
+        :param stop_words: Up to 4 sequences where the API stops generating further tokens. The returned text does
```

Comment:
what's a "sequence"? is it a string? If yes, I think it's clearer to say:
Specifies the strings that make the API stop generating more tokens. The returned text doesn't contain these strings. You can specify up to four such strings.